doc/user/sql/tail.md

```diff
@@ -36,6 +37,15 @@ Field | Represents
 `diff value` | Whether the record is an insert (`1`), delete (`-1`), or update (delete for old value, followed by insert of new value).
 `logical timestamp` | Materialize's internal logical timestamp.
 
+### AS OF
+
+If a timestamp is explicitly requested via `AS OF`, that timestamp will be used to determine what events to report rather than one automatically determined by Materialize.
```

Comment:
Pardon my English-teacher hat here! If we converted this sentence to active voice, I think we could make it a little clearer what users should expect, e.g. framing it from the user's perspective.

> You can choose a point in time to begin reporting events using `AS OF`...

Not saying it should be _that_ verbatim, but something akin to it.

Reply:
Or alternately (I <3 active voice, but I prefer the subject to be the actor we are explaining).

> Materialize will use the timestamp requested with `AS OF` to determine precisely which version of the collection to report. Materialize will automatically determine an appropriate timestamp if `AS OF` is not used.

General theory here is that folks are reading the docs to learn a bit about the process by which things happen, and telling them that from the POV of the thing doing it works great! 

But no strong feels!

Reply:
IME there's a bifurcation between "conceptual" and "task-oriented" docs. Conceptual docs want the components POV; task-oriented docs should be written from the users' POV. The SQL docs are often more task-oriented because people often use them as a quick reference to do something quickly. To address the conceptual components, most of the docs have a Conceptual Framework section where it's more appropriate the make the components the actors.

Just my 2 cents, though, and these things often depend on a number of factors.